.github/workflows/main.yml

```
@@ -38,7 +38,7 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
       - name: Install Ansible
-        run: pip install git+https://github.com/ansible/ansible.git@${{ matrix.ansible }}#egg=ansible
+        run: pip install git+https://github.com/ansible/ansible.git@${{ matrix.ansible }}
```

How about:
```suggestion
        run: pip install git+https://github.com/ansible/ansible.git@${{ matrix.ansible }}#egg=ansible-base
```

according to https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support the egg is needed for dependency resolution before actually downloading anything. That could sure be a problem...

---

It's only called ansible-base in devel, so that'd break the stable branches. And nothing depends on ansible in our requirements, so it should be fine. Emphasis on should.